plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -127,6 +129,22 @@ public PluginManager(Config cfg) {
         skipFailedPlugins = cfg.isSkipFailedPlugins();
         hashFunction = cfg.getHashFunction();
         httpClient = null;
+        userAgentInformation = this.getUserAgentInformation();
+    }
+
+    private String getUserAgentInformation() {
+        String userAgentInformation= "Jenkins plugin manager/Dev";
+        Properties properties = new Properties();
+        try (InputStream propertiesStream = this.getClass().getClassLoader().getResourceAsStream("version.properties")) {
```

Comment:
`mvn clean install` fails for me locally:

```
[INFO] Total bugs: 1
[ERROR] Medium: Redundant nullcheck of propertiesStream, which is known to be non-null in io.jenkins.tools.pluginmanager.impl.PluginManager.getUserAgentInformation() [io.jenkins.tools.pluginmanager.impl.PluginManager] Redundant null check at PluginManager.java:[line 138] RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE
[INFO]

```

Reply:
It passed for me for all the changes mentioned above, I will try again. I only committed changes if they passed the tests. Did the latest change break it? Will have to check it. will update you asap

Reply:
I think the error is because I have used an `if (propertiesStream!=null)` after `try` statement. I guess removing it will fix this. On my system this was not detected idk why. I will send the commit.